src/tensor_ops/matmul.rs

```diff
@@ -311,6 +411,47 @@ fn mm_atct<const M: usize, const K: usize, const N: usize>(
     }
 }
 
+/// batch matrix multiply `c += a * b`
+fn bmm<const M: usize, const K: usize, const N: usize, const B: usize>(
+    a: &[[[f32; K]; M]; B],
+    b: &[[f32; N]; K],
+    c: &mut [[[f32; N]; M]; B],
+) {
+    let b = b.as_ptr() as *const f32;
+
+    // Purely sequential for now, should parallelize using rayon or some other BLAS solution
+    for i in 0..B {
+        let a = a[i].as_ptr() as *const f32;
+        let c = c[i].as_mut_ptr() as *mut f32;
+
+        unsafe {
+            matrixmultiply::sgemm(
```

Comment:
We'll need to add MKL support (that's the cblas calls) for `bmm` and `bmm_bt`. That can be a separate PR though, for now we can juts open an issue to track it.